core/server/src/main/webapp/header.jsp

```diff
@@ -1,26 +1,34 @@
-      <li><a href="http://alluxio.org/" target="_blank"><img style="height:25px;margin-top:-5px;" src="img/logo.png" alt="Alluxio Logo"/></a></li>
-      <li id="home-li"><a href="./home">Overview</a></li>
-      <% if (request.getAttribute("useWorkerHeader") == null) { %>
-        <li id="browse-li"><a href="./browse?path=/">Browse File System</a></li>
-        <li id="configuration-li"><a href="./configuration">System Configuration</a></li>
-        <li id="workers-li"><a href="./workers">Workers</a></li>
-        <li id="memory-li"><a href="./memory">In-Memory Files</a></li>
-      <% } else {%>
-        <li id="blockInfo-li"><a href="./blockInfo">BlockInfo</a></li>
-      <% } %>
-      <li id="browseLogs-li"><a href="./browseLogs">Log Files</a></li>
-      <li id="autorefresh-li"><a href="javascript:toggleAutoRefresh();" id="autorefresh-link">Enable Auto-Refresh</a></li>
-      <li id="metricsui-li"><a href="./metricsui">Metrics</a></li>
-      <% if (request.getAttribute("useWorkerHeader") != null) { %>
-        <li id="returnmaster-li"><a href="http://<%=request.getAttribute("masterHost") %>:<%= request.getAttribute("masterPort") %>">Return to Master</a></li>
-      <% } %>
-    </ul>
+    <a class="btn btn-navbar" data-toggle="collapse" data-target=".nav-collapse">
+      <span class="icon-bar"></span>
+      <span class="icon-bar"></span>
+      <span class="icon-bar"></span>
+    </a>
+    <a href="http://alluxio.org/" target="_blank" class="brand" ><img style="height:25px;margin-top:-5px;" src="img/logo.png" alt="Alluxio Logo"/></a>
+    <div class="nav-collapse collapse">
+      <ul class="nav nav-pills">
+        <li id="home-li"><a href="./home">Overview</a></li>
+        <% if (request.getAttribute("useWorkerHeader") == null) { %>
+          <li id="browse-li"><a href="./browse?path=/">Browse File System</a></li>
+          <li id="configuration-li"><a href="./configuration">System Configuration</a></li>
+          <li id="workers-li"><a href="./workers">Workers</a></li>
+          <li id="memory-li"><a href="./memory">In-Memory Files</a></li>
+        <% } else {%>
+          <li id="blockInfo-li"><a href="./blockInfo">BlockInfo</a></li>
+        <% } %>
+        <li id="browseLogs-li"><a href="./browseLogs">Log Files</a></li>
+        <li id="autorefresh-li"><a href="javascript:toggleAutoRefresh();" id="autorefresh-link">Enable Auto-Refresh</a></li>
+        <li id="metricsui-li"><a href="./metricsui">Metrics</a></li>
+        <% if (request.getAttribute("useWorkerHeader") != null) { %>
+          <li id="returnmaster-li"><a href="http://<%=request.getAttribute("masterHost") %>:<%= request.getAttribute("masterPort") %>">Return to Master</a></li>
+        <% } %>
+      </ul>
+    </div>
   </div>
 </div>
 <script>
   var url = window.location.pathname;
   $("#"+url.substring(1)+"-li").addClass("active");
 </script>
+<link rel="stylesheet" type="text/css" href="css/bootstrap-responsive.min.css">
```

Comment:
@xxhuang yes, for other css imports, they are put in the header blocks. But if it is put in webapp/general.jsp, I think it also needs to be put in [webapp/configuration.jsp](https://github.com/Alluxio/alluxio/blob/master/core/server/src/main/webapp/configuration.jsp#L18) etc.